content/chat/reactions.textile

```diff
@@ -5,3 +5,138 @@ product: chat
 languages:
   - javascript
 ---
+
+In most chat environments you will have encountered some form of message reactions. The ability to add a reaction such as an emoji to a fellow chat participant's message is an important piece of functionality for users to express their opinion without replying with text.
```

Comment:
I think we need to tidy up all of the page intros to be a little more succinct and use the active voice. E.g.:

"React to previously sent messages with emojis such as a thumbs up, or a heart. Reactions enable users to quickly express their opinion without having to type."

Reply:
I've updated the intros on the page. I hope they're more succinct, and with an active voice.